test/extensions/transport_sockets/tls/test_data/bad_rsa_key_usage_cert.cfg

```diff
@@ -0,0 +1,38 @@
+[req]
```

Comment:
Please add a comment in this file indicating exactly which field causes this to be an incorrect key usage.